K2Bridge/Visitors/Aggregations/MetricAggregationVisitor.cs

```diff
@@ -33,6 +34,29 @@ public void Visit(CardinalityAggregation cardinalityAggregation)
             cardinalityAggregation.KustoQL = $"{EncodeKustoField(cardinalityAggregation.Key)}={KustoQLOperators.DCount}({EncodeKustoField(cardinalityAggregation)})";
         }
 
+        /// <inheritdoc/>
+        public void Visit(ExtendedStatsAggregation extendedStatsAggregation)
+        {
+            Ensure.IsNotNull(extendedStatsAggregation, nameof(extendedStatsAggregation));
+            EnsureClause.StringIsNotNullOrEmpty(extendedStatsAggregation.Field, extendedStatsAggregation.Field, ExceptionMessage);
+
+            var sep = AggregationsConstants.MetadataSeparator;
+            var key = $"['{extendedStatsAggregation.Key}{sep}extended_stats']";
+            var average = $"{KustoQLOperators.Avg}({EncodeKustoField(extendedStatsAggregation)})";
+            var standard_deviation = $"{KustoQLOperators.StDev}({EncodeKustoField(extendedStatsAggregation)})";
+
+            var query = new StringBuilder();
+
+            query.Append($"{key}={KustoQLOperators.Pack}(");
```

Comment:
Is there a good reason to use the array pack trick? why aren't they just normal columns?

Reply:
Because we need to have all the information in the same key (like `['1']` or `['2']` and so on )
All these values belongs to the standard deviation.
It's more convenient to have just one column when you deserialize everything

Reply:
I see your point @eladiw @tomconte @dupuyjs @nebrass  WDYT?

Reply:
No objection for this trick. If you consider percentiles scenario, percentiles_array returns all values in the same column. Here, this is the same concept. The only potential issue will be to lost the associated data type (as we now have a json). But here all values are double and pack everything in one only column really help during response parsing.

Reply:
Ok, closing